aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -10,6 +10,7 @@
 #include <ATen/native/cpu/zmath.h>
 #include <ATen/Parallel.h>
 
+#include <torch/csrc/autograd/functions/utils.h>
```

Comment:
I'm afraid you can't do that... libtorch depends on aten, but not the other way around...

Reply:
I guess you can work around this here by checking `GradMode::is_enabled() && input.requires_grad()` below instead of using this function...

Reply:
Alright, I will change it then.

Reply:
@albanD, done.